tools-libselinux/pkg.yaml

```diff
@@ -30,6 +30,13 @@ steps:
     test:
       - |
         fhs-validator /rootfs
+    sbom:
+      outputPath: /rootfs/usr/share/spdx/libselinux.spdx.json
+      version: {{ .selinux_version }}
+      cpes:
+        - cpe:2.3:a:selinuxproject:libselinux:{{ .selinux_version }}:*:*:*:*:*:*:*
+      licenses:
+        - CUSTOM
```

Comment:
Public domain, which does not have an SPDX ID: https://github.com/SELinuxProject/selinux/blob/main/libselinux/LICENSE